src/index.js

```diff
@@ -0,0 +1,169 @@
+import './styles.css';
+
+const todoList = document.getElementById('todo-list');
+
+function saveData() {
+  localStorage.setItem('data', todoList.innerHTML);
```

Comment:
- [ ] The index should be updated when deleting elements, currently it does not update on your app. Also it would be more efficient if local-storage would store only the array with the todo name and its index instead of the whole html